pkg/webui/console/components/payload-formatters-form/index.js

```diff
@@ -197,13 +204,46 @@ class PayloadFormattersForm extends React.Component {
     }
   }
 
+  @bind
+  pastePayloadFormatter(app) {
+    const { defaultParameter, uplink, repoFormatters } = this.props
+    const applicationFormatter = defaultParameter
+      ? defaultParameter
+      : getDefaultJavascriptFormatter(uplink)
```

Comment:
```suggestion
    const applicationFormatter = defaultParameter || getDefaultJavascriptFormatter(uplink)
```

You can just use a logical OR to achieve the same thing.